javascript/javascript1/week3/TripTime.js

```diff
@@ -0,0 +1,30 @@
+//When will we be there??
+const travelInformation = {
+  speed: 50,
+  destinationDistance: 432
+};
+
+const travelTimeInMinutes = getTimeOfTrip(travelInformation);
+console.log("InMinutes:" + travelTimeInMinutes); 
+
+let hoursAndMinutes = timeConvert(travelTimeInMinutes);
+console.log(travelTimeInMinutes + " minutes = " + hoursAndMinutes);
+
+//Time = Distance ÷ Speed
+function getTimeOfTrip(travelInformation) {
+  let time = (travelInformation.destinationDistance / travelInformation.speed) * 60;
+
+  return time;
+}
+
+function timeConvert(timeInMinutes) {
+  let fractionalHours = timeInMinutes / 60;
+  let hours = Math.floor(fractionalHours);
+  let fractionalMinutes = (fractionalHours - hours) * 60;
+  let minutes = Math.round(fractionalMinutes);
+  let hoursAndMinutes = hours + " hour(s) and " + minutes + " minute(s)."
```

Comment:
it's probably better to return an object {hours, minutes} than a string with formatted output. You can do the string formatting outside.